client/my-sites/checkout/composite-checkout/hooks/product-variants.tsx

```diff
@@ -126,6 +126,9 @@ export function useGetProductVariants(
 					? variant.introductoryOfferPrice
 					: variant.priceFinal || variant.priceFull;
 
+			const regularPrice = variant.priceFull ? variant.priceFull : price;
+			const introIntervalCount = variant.product.introductory_offer?.interval_count ?? 0;
+			const introIntervalUnit = variant.product.introductory_offer?.interval_unit ?? 'month';
```

Comment:
If the default `month` value is used as is in a string, it should be translatable.